vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -39,8 +39,8 @@ switch -Regex ($windowsSku) {
         )
     }
     "2022-containerd*" {
-        $global:patchUrls = @("https://catalog.s.download.windowsupdate.com/d/msdownload/update/software/secu/2024/03/windows10.0-kb5035857-x64_f699534e61e7e8e750a7f751f6a1ff4d03bd3ebb.msu")
```

Comment:
why removing these two lines?

Reply:
Used for test. Will add them back after test.

Reply:
Please add them back